stackslib/src/chainstate/nakamoto/signer_set.rs

```diff
@@ -261,22 +289,35 @@ impl NakamotoSigners {
             );
         }
 
-        let args = [
-            SymbolicExpression::atom_value(Value::cons_list_unsanitized(signers_list).expect(
+        let set_stackerdb_args = [
```

Comment:
nit: naming should be `set_signer_slots_args`.

Reply:
I think that `signer_slots` is part of the confusion here. `stackerdb_slots` and `signer_weights` aren't the same thing -- the function it is calling is setting the stackerdb config for `.signers`, the other function is setting the signer's weights.